configs/sites/tier1/acorn/packages_intel.yaml

```diff
@@ -41,11 +44,16 @@
       require:
       - any_of: ["@2.27.0"]
         when: "%intel@19.1.3.304"
+        message: "2.25.0 is the last version to use C++11 (as opposed to C++17)"
```

Comment:
Is 2.25.0 not required now?  Was the C++ standard updated to 17 with this update?

Reply:
2.27.0 is no longer in the recipe, so I'm leaving this as 2.25.0 (which is what's now installed for 1.8.0)